libraries/abstractions/common_io/test/test_iot_spi.c

```diff
@@ -312,8 +312,10 @@ TEST( TEST_IOT_SPI, AFQP_IotSPI_ReadSyncAssisted )
 
     /* restore slave select */
     if( ulAssistedTestIotSpiSlave!=ultestIotSpiSlave ){
+#ifdef ASSISTED_TEST_COMMON_IO_SPI_SLAVE_SELECET_SUPPORTED
```

Comment:
Can you rename this macro to "IOT_TEST_COMMON_IO_SPI_SLAVE_SELECET_SUPPORTED

Reply:
Also, select appears misspelled – can we correct the spelling while we're at it?